extensions/amp-form/0.1/form-dirtiness.js

```diff
@@ -70,10 +78,19 @@ export class FormDirtiness {
    */
   onSubmitSuccess() {
     this.isSubmitting_ = false;
+    this.submittedFormData_ = this.takeFormDataSnapshot_();
```

Comment:
I chose to ignore the "possible changes to the form between `onSubmitting` and `onSubmitSuccess`" edge case for now, objections? @GoTcWang

Reply:
It's okay we leave that for now. But that's definitely something we should figure out, even on version 1.
As long as you have considered that "corner case" in advance, feel free to add a TODO here and resolve it in another PR.